build_tools/run.py

```diff
@@ -32,11 +32,16 @@ def _to_str(x):
     return x
 
 def _exec_command(arg_list, **kwargs):
+    silent = False
```

Comment:
We need to be able to sometimes run a command without printing the output to the console. Example is when printing the authentication token.

Reply:
is it because it prints secrest?

Reply:
Correct. We start by getting the service account key from a GH secret and activate the service account. This part is fine, but in the next step we call `gcloud auth print-access-token` and capture the output (short lived access token) and use that in the subsequent signing step. We want to capture the token but not print it to stdout.